layouts/partials/posts/math.html

```diff
@@ -1,4 +1,4 @@
-{{- if eq .Params.math "true" -}}
+{{- if (or (eq .Params.math true) (eq .Params.math "true")) -}}
```

Comment:
```suggestion
{{- if .Params.math -}}
```
I think this is a better way to handle the problem.

Reply:
@luizdepra of course it is better lol :D; I just assumed that `true != "true"` in the conditional context.

Thank you.